app/components/bali/tags/component.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Bali
+  module Tags
+    class Component < ApplicationViewComponent
+      renders_many :tag_items, ->(**options) do
+        size = @all_sizes || nil
+        TagItem::Component.new(
+          is_light: @all_light,
+          rounded: @all_rounded,
+          size: size,
+          **options
+        )
+      end
+
+      def initialize(
+        sizes: nil,
+        all_light: false,
+        all_rounded: false,
```

Comment:
aqui le podemos simplificar el nombre

```suggestion
        light: false,
        rounded: false,
```